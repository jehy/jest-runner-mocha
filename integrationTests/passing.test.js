const runJest = require('./runJest');

it('Works when it has only passing tests', () => {
  return expect(runJest('passing')).resolves.toMatchSnapshot();
});

it('Works when it has only passing tests and --coverage', () => {
  return expect(runJest('passing', ['--coverage'])).resolves.toMatchSnapshot();
});

it('Works when it has only passing tests and --coverage with v8', () => {
  return expect(
    runJest('passing', ['--coverage', '--coverageProvider', 'v8']),
  ).resolves.toMatchSnapshot();
});
