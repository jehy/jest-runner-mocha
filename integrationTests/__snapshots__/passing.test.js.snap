// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has only passing tests 1`] = `
"[0m[7m[1m[32m PASS [39m[22m[27m[0m [2mintegrationTests/__fixtures__/passing/__mocha__/__tests__/[22m[1mfile.test.js[22m
  My tests
    [32mâœ“[39m [2mThis test passes[22m

[1mTest Suites:[22m[1m[32m1 passed[39m[22m, 1 total
[1mTests:      [22m[1m[32m1 passed[39m[22m, 1 total
[1mSnapshots:  [22m0 total
[1mTime:[22m        
[2mRan all test suites[22m[2m.[22m
"
`;

exports[`Works when it has only passing tests and --coverage 1`] = `
"[0m[7m[1m[32m PASS [39m[22m[27m[0m [2mintegrationTests/__fixtures__/passing/__mocha__/__tests__/[22m[1mfile.test.js[22m
  My tests
    [32mâœ“[39m [2mThis test passes[22m

[1mTest Suites:[22m[1m[32m1 passed[39m[22m, 1 total
[1mTests:      [22m[1m[32m1 passed[39m[22m, 1 total
[1mSnapshots:  [22m0 total
[1mTime:[22m        
[2mRan all test suites[22m[2m.[22m
--------------|---------|----------|---------|---------|-------------------
File          | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
--------------|---------|----------|---------|---------|-------------------
[32;1mAll files    [0m | [32;1m    100[0m | [32;1m     100[0m | [32;1m    100[0m | [32;1m    100[0m | [33;1m                 [0m 
[32;1m file.test.js[0m | [32;1m    100[0m | [32;1m     100[0m | [32;1m    100[0m | [32;1m    100[0m | [33;1m                 [0m 
--------------|---------|----------|---------|---------|-------------------"
`;
